test/__mocks__/session.mock.ts

```diff
@@ -1064,7 +1064,7 @@ export const UPDATE_MANAGING_OFFICER_SINGLE_NATIONALITY_MOCK: managingOfficerTyp
   last_name: "Wabb",
   has_former_names: yesNoResponse.Yes,
   former_names: "Jimmothy James Jimminny, Finn McCumhaill, Test Tester",
-  date_of_birth: { day: "1", month: "02", year: "1900" },
+  date_of_birth: { day: "01", month: "02", year: "1900" },
```

Comment:
could you add some tests for mapDateOfBirth into test/utils/update/mapper.utils.spec.ts to test some DOB's with single digit, double digit number days and also with day as a single digit and double digit string?
I think the previous authors of that function should have added some tests into that test file.

Reply:
Tests added